modules/templates/global-elements.xml

```diff
@@ -485,4 +485,155 @@ $('.date').each(function(_idx, dateEl){
             <asset id="image" type="path" cmsOnly="true" mimeType="image/png" path="/modules/assets/template-thumbnails/global/image.png" />
         </assets>
     </template>
+    <template>
+        <id>line</id>
+        <title>Line</title>
+        <type>element</type>
+        <dataType>global</dataType>
+        <thumbnail>square-thumb</thumbnail>
```

Comment:
We need to change the thumbnail for this element, I will send you one ( that might be temporary ) just we can easily distinguish between Line and Square in the toolbar